### DISCUSSION TITLE
Master and Multiple Slave SX1262

### DISCUSSION THREAD

**Discussion by NithinAravindKP** [jgromes/RadioLib#797] [Q&A]
**IMPORTANT: Check the wiki**
Before submitting new issue, please check the [Wiki](https://github.com/jgromes/RadioLib/wiki) and the [API documentation](https://jgromes.github.io/RadioLib/). You might find a solution to your issue there. 

**Is your feature request related to a problem? Please describe.**
I am trying to configure Master slave concept using ESP32+LoRa SX1262 modules. But Not able to configure this

**Describe the solution you'd like**
I would like to get a working code which has Master node sending data to Slaves on interrupt or event based. But master will be always looking/receive for Slave input

**Describe alternatives you've considered**

**Additional context**
I am trying to modify your SX126x_Transmit_Interrupt and SX126x_Receivet_Interrupt C code. Combing the Tx and Rx code in master node, always works as Tx not Rx/TX as expected

**Comment by jgromes** [comment]
> I am trying to modify your SX126x_Transmit_Interrupt and SX126x_Receivet_Interrupt C code

Have you tried the PingPong example?

> I would like to get a working code which has Master node sending data to Slaves on interrupt or event based. But master will be always looking/receive for Slave input

You will need a way to synchronize the replies from slaves - if two of them attempt to transmit at the same time, it is likely the master node will not receive anything.

Also, not an issue (and definitely not a feature request, since the SX126x has all the functionality required to implement what you're looking for), so I'm moving this to discussions.

**Reply by NithinAravindKP** [reply]
Yes i have tried the Ping Pong example, which was not meeting my  requirement. 

Yes i am aware of two slaves sending data at the same time, planning for a header concept to filter it only the required the slave communication at a given time.

Appreciate your support for moving it to a discussion.
